ui/frontend/index.ejs

```diff
@@ -1,11 +1,12 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <meta name="description" content="UI for the Rust playground" />
```

Comment:
This doesn’t feel like a *useful* description. Is there Some way we could describe the purpose of the playground or the kinds of things it would be used for?

Reply:
yeah, I just grabbed this from the `package.json`...how about something like:
"A full-featured Rust runtime in the browser to experiment with any part of the language that you want"

...but I'm totally happy to just put in whatever you think makes the most sense.